inst/tests/tests.Rraw

```diff
@@ -18330,3 +18330,22 @@ if (test_bit64) {
   apple = data.table(id = c("a", "b", "b"), time = c(1L, 1L, 2L), y = i64v[1:3])
   test(2248, dcast(apple, id ~ time, value.var = "y"), data.table(id = c('a', 'b'), `1` = i64v[1:2], `2` = i64v[4:3], key='id'))
 }
+
+# fread(...,fill) can also be used to specify a guess on the maximum number of columns #2691 #1812 #4130 #3436 #2727
```

Comment:
I see one example here derived from the closing issues, but I think it'd be better if we included more of those as specific regression tests before closing the linked issue.